@xen-orchestra/backups/_cleanVm.mjs

```diff
@@ -7,13 +7,10 @@ import { basename, dirname, resolve } from 'node:path'
 import { isMetadataFile, isVhdFile, isXvaFile, isXvaSumFile } from './_backupType.mjs'
 import { limitConcurrency } from 'limit-concurrency-decorator'
 import { mergeVhdChain } from 'vhd-lib/merge.js'
-
 import { Task } from './Task.mjs'
 import { Disposable } from 'promise-toolbox'
-import handlerPath from '@xen-orchestra/fs/path'
-
+import * as handlerPath from '@xen-orchestra/fs/path'
```

Comment:
I am not sure to understand why we have both the .mjs and .mts file . Is the .mjs built from the .mts ?